src/app/services/auth/auth.service.spec.ts

```diff
@@ -26,4 +27,18 @@ describe('AuthService', () => {
       expect(service.logout).toBeTruthy();
     }
   ));
+
+    it('should login', inject([AuthService], (service: AuthService) => {
+        service.getToken('admin', 'admin').subscribe(() => {
+            expect(service.isLoggedIn()).toEqual(true);
+        });
+    }));
+
+    it('should fail on invalid login', inject([AuthService], (service: AuthService) => {
+        service.getToken('admin2', 'admin').subscribe(() => {
+            // NOP
```

Comment:
+1

Reply:
Hopefully nobody thinks this means nope x)